src/Provider/AbstractProvider.php

```diff
@@ -126,7 +138,7 @@ public function __construct(array $options = [], array $collaborators = [])
      * Sets the grant factory instance.
      *
      * @param  GrantFactory $factory
-     * @return $this
+     * @return self
```

Comment:
I'm not sure what the standard is, but if you use `$this`, the generated documentation just contains the string "$this." If you use `self`, the generated documentation provides a link to the current class.

Of course, this could differ from one implementation to another, because there is no standard. I've been using ApiGen to generate the docs. :-)